Core/include/Acts/Clusterization/Clusterization.ipp

```diff
@@ -308,9 +309,7 @@ ClusterCollection mergeClusters(CellCollection& cells) {
   if constexpr (GridDim > 1) {
     // Sort the cells by their cluster label, only needed if more than
     // one spatial dimension
-    std::sort(cells.begin(), cells.end(), [](Cell& lhs, Cell& rhs) {
-      return getCellLabel(lhs) < getCellLabel(rhs);
-    });
+    std::ranges::sort(cells, {}, [](Cell& c) { return getCellLabel(c); });
```

Comment:
Can't this just be written via η-reduction:

```suggestion
    std::ranges::sort(cells, {}, getCellLabel);
```

Reply:
I tried, but somehow it doesn't work. I also tried to specify the namespace. At other places I can to a reduction for things like `x.first` and it works.

```
In file included from <source>/Core/include/Acts/Clusterization/Clusterization.hpp:98,
                 from <source>/Examples/Algorithms/Digitization/include/ActsExamples/Digitization/ModuleClusters.hpp:11,
                 from <source>/Examples/Algorithms/Digitization/src/ModuleClusters.cpp:9:
<source>/Core/include/Acts/Clusterization/Clusterization.ipp: In function ‘ClusterCollection Acts::Ccl::mergeClusters(CellCollection&)’:
<source>/Core/include/Acts/Clusterization/Clusterization.ipp:312:34: error: ‘getCellLabel’ was not declared in this scope
  312 |     std::ranges::sort(cells, {}, getCellLabel);
      |                                  ^~~~~~~~~~~~
```

Reply:
That doesn't make any sense to me, but I guess it's also fine to keep it like you have it now. :)